File: 0001-two-sum/0001-two-sum.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        let mut solution_vect = vec![];

        for i in 0..nums.len() {
            for j in (i + 1)..nums.len() {
                if nums[i] + nums[j] == target {
                    solution_vect.push(i as i32);
                    solution_vect.push(j as i32);
                }
            }
        }

        solution_vect
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        idx = 0
        idx_2 = 1
        length = len(nums)
        while length>0:
            x = target - nums[idx]
            if x in nums[idx_2:]:
                idx_2 = nums.index(x, idx + 1)
                break
            else:
                idx = idx+1
                idx_2 = idx_2+1
                length = length-1
        return (idx, idx_2)


        